tests/keyboard-test/UbuntuKeyboard/Dummy.qml

```diff
@@ -1,5 +0,0 @@
-import QtQuick 2.0
-
-Item {
-    property bool available: true
-}
```

Comment:
The corresponding C++ plugin is implemented by src/plugin/plugin.cpp ; my idea was that this code, today empty, could maybe prove itself useful at some point in time.